src/index.css

```diff
@@ -33,3 +33,16 @@ body {
   width: 100%;
   max-width: 800px;
 }
+
+.fade-in {
```

Comment:
What's this for? I see where its used in the code but can't tell what it is doing

Reply:
It's making the tab content fade in when we switch routes. We were using a bootstrap class to do this before but it wasn't working with the way that I changed the routing so I had to add it as an animation